L1Trigger/RPCTechnicalTrigger/src/RBCProcessRPCDigis.cc

```diff
@@ -363,7 +272,7 @@ void RBCProcessRPCDigis::setDigiAt( int sector, int digipos )
 void RBCProcessRPCDigis::setInputBit( std::bitset<15> & signals , int digipos ) 
 {
   
-  int bitpos = m_layermap[digipos];
+  int bitpos = s_layermap.at(digipos);
```

Comment:
This is a functional differerence. If an out of range `digipos` is used, this code will now throw. The old code just added a new entry.